capture/genBitmaps.js

```diff
@@ -12,12 +11,25 @@ if (!config.paths) {
   config.paths = {};
 }
 
+String.prototype.hashCode = function () {
```

Comment:
What is the reason for adding this to the string prototype as opposed to just a local method?